examples/Example/src/touchables/index.tsx

```diff
@@ -241,67 +255,83 @@ const TOUCHABLES = [
 
   {
     type: TouchableNativeFeedback,
-    background: A => A.SelectableBackground(),
+    background: (A) => A.SelectableBackground(),
     color: 'transparent',
     renderChild: renderSampleBox,
     text: 'TouchableNativeFeedback (SelectableBackground) tranparent',
   },
   {
     type: TouchableNativeFeedback,
-    background: A => A.SelectableBackgroundBorderless(),
+    background: (A) => A.SelectableBackgroundBorderless(),
     color: 'honeydew',
     renderChild: renderSampleBox,
     text: 'TouchableNativeFeedback (SelectableBackgroundBorderless)',
   },
   {
     type: TouchableNativeFeedback,
-    background: A => A.Ripple('floralwhite', true),
+    background: (A) => A.Ripple('floralwhite', true),
     color: 'greenyellow',
     renderChild: renderSampleBox,
     text: 'TouchableNativeFeedback (Ripple, borderless: true)',
   },
   {
     type: TouchableNativeFeedback,
-    background: A => A.Ripple('blue', true, 30),
+    // the arguments passed to A.Ripple look like they are taken from PressableAndroidRippleConfig, where the radius is present
+    // but the TouchableNativeFeedback exported from RNGH is the same as from RN and none it does not have this interface included
+    // in the static Ripple method. But it somehow works.
+    // I looks like a mistake. I submitted a PR with the change: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50063
+    // @ts-ignore it is no-op because of the types
+    background: (A) => A.Ripple('blue', true, 30),
     color: 'green',
     renderChild: renderSampleBox,
     text: 'TouchableNativeFeedback (Ripple, borderless: true, radius: 30)',
   },
   {
     type: TouchableNativeFeedback,
-    background: A => A.Ripple('darkslategrey', false),
+    background: (A) => A.Ripple('darkslategrey', false),
     color: 'dodgerblue',
     renderChild: renderSampleBox,
     text: 'TouchableNativeFeedback (Ripple, borderless: false)',
   },
 ];
 
-const screens = TOUCHABLES.reduce(
-  (map, obj) => ((map[obj.text] = obj), map),
+const screens: Record<string, any> = TOUCHABLES.reduce(
```

Comment:
`any` === `TouchablesType` from reduce function context.

Reply:
Will check and change if it can be changed.